com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -856,7 +856,7 @@ private void NetworkUpdate()
         /// </summary>
         private void NetworkLateUpdate()
         {
-            RpcQueueManager?.ProcessAndFlushRPCQueue(RPCQueueManager.RPCQueueProcessingTypes.Send);
+            RpcQueueManager?.ProcessAndFlushRPCQueue(RpcQueueContainer.RPCQueueProcessingTypes.Send);
```

Comment:
probably no need for `?.` here too

Reply:
This will be fixed in the NetworkUpdate feature.